lib/experimental/fonts/font-library/class-wp-rest-font-families-controller.php

```diff
@@ -18,81 +18,12 @@
  */
 class WP_REST_Font_Families_Controller extends WP_REST_Posts_Controller {
 	/**
-	 * Constructor.
+	 * Whether the controller supports batching.
 	 *
 	 * @since 6.5.0
+	 * @var false
 	 */
-	public function __construct() {
-		$post_type       = 'wp_font_family';
-		$this->post_type = $post_type;
-
-		$post_type_obj   = get_post_type_object( $post_type );
-		$this->rest_base = $post_type_obj->rest_base;
-		$this->namespace = $post_type_obj->rest_namespace;
-	}
-
-	/**
-	 * Registers the routes for the objects of the controller.
-	 *
-	 * @since 6.5.0
-	 */
-	public function register_routes() {
-		register_rest_route(
-			$this->namespace,
-			'/' . $this->rest_base,
-			array(
-				array(
-					'methods'             => WP_REST_Server::READABLE,
-					'callback'            => array( $this, 'get_items' ),
-					'permission_callback' => array( $this, 'get_font_families_permissions_check' ),
-					'args'                => $this->get_collection_params(),
-				),
-				array(
-					'methods'             => WP_REST_Server::CREATABLE,
-					'callback'            => array( $this, 'create_item' ),
-					'permission_callback' => array( $this, 'create_item_permissions_check' ),
-					'args'                => $this->get_create_edit_params(),
-				),
-				'schema' => array( $this, 'get_public_item_schema' ),
-			)
-		);
-
-		register_rest_route(
-			$this->namespace,
-			'/' . $this->rest_base . '/(?P<id>[\d]+)',
-			array(
-				'id' => array(
-					'description' => __( 'Unique identifier for the font family.', 'gutenberg' ),
-					'type'        => 'integer',
-					'required'    => true,
-				),
-				array(
-					'methods'             => WP_REST_Server::READABLE,
-					'callback'            => array( $this, 'get_item' ),
-					'permission_callback' => array( $this, 'get_font_families_permissions_check' ),
-					'args'                => array(),
-				),
-				array(
-					'methods'             => WP_REST_Server::EDITABLE,
-					'callback'            => array( $this, 'update_item' ),
-					'permission_callback' => array( $this, 'update_item_permissions_check' ),
-					'args'                => $this->get_create_edit_params(),
-				),
-				array(
-					'methods'             => WP_REST_Server::DELETABLE,
-					'callback'            => array( $this, 'delete_item' ),
-					'permission_callback' => array( $this, 'delete_item_permissions_check' ),
-					'args'                => array(
-						'force' => array(
-							'type'        => 'boolean',
-							'default'     => false,
-							'description' => __( 'Whether to bypass Trash and force deletion.', 'default' ),
-						),
-					),
-				),
-			)
-		);
-	}
```

Comment:
Removing the constructor and register_routes methods to depend on the parents, as requested.